docker-compose.yaml

```diff
@@ -15,7 +15,7 @@ services:
     environment:
```

Comment:
I can't highlight the first line, but I believe `version` can be removed as it is now obsolete, as indicated by the logs here:

```
$ docker compose up
...
WARN[0000] /home/ec2-user/SynapseWorkflowOrchestrator/docker-compose.yaml: `version` is obsolete 
```